e2e-test/peerforwarder/build.gradle

```diff
@@ -174,6 +176,17 @@ def createLocalAggregateDataPrepper2Task = createDataPrepperDockerContainer(
 def localAggregateEndToEndTest = createEndToEndTest("localAggregateEndToEndTest", includeLocalAggregateTestsMatchPattern,
         createLocalAggregateDataPrepper1Task, createLocalAggregateDataPrepper2Task)
 
+// Discovery mode: LOCAL_NODE
```

Comment:
minor: Can you add more context here that we are testing metrics using LOCAL NODE?